ironfish/src/logger/index.ts

```diff
@@ -8,7 +8,13 @@ import { parseLogLevelConfig } from './logLevelParser'
 import { ConsoleReporter } from './reporters/console'
 export * from './reporters/intercept'
 
-export type Logger = Consola
+type Loggable = string | number
+export interface Logger extends Consola {
+  info(message: string, args?: Record<string, Loggable>): void
+  error(message: string, args?: Record<string, Loggable>): void
+  warn(message: string, args?: Record<string, Loggable>): void
+  debug(message: string, args?: Record<string, Loggable>): void
+}
```

Comment:
It doesn't look like there are many places where we haven't conformed to this interface in the past, but is there any concern over making the logging interface more restrictive to use?

Reply:
Yea just judging from our past usage I think this is not too big of a deal and it would force people to think about if it's really worth it to have something less structured in the logs. I'll add some comments here so that there is some explanation as to why it is restrictive. If we need to make it less restrictive in the future I think we can cross that bridge when we come to it